ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/VulnerabilityStateTabs.tsx

```diff
@@ -32,7 +32,7 @@ function VulnerabilityStateTabs({
             activeKey={vulnerabilityStateKey}
             onSelect={(_e, tab) => {
                 setVulnerabilityStateKey(tab);
-                if (onChange && isVulnerabilityState(tab)) {
+                if (onChange && isVulnerabilityState(tab) && tab !== vulnerabilityStateKey) {
```

Comment:
Prevents firing the `onChange` event callback if the selected tab is the current tab.